### ISSUE TITLE
As a developer, I want to understand the team's approach to styling the frontend

### ISSUE THREAD

**Issue by TangoYankee** [openoakland/woeip#92]
**Is your feature request related to a problem? Please describe.**
I would like to know the guidelines for styling elements on the website

**Describe the solution you'd like**
I would like the styling to default to the Foundation built-in styling, for the MVP.
- It requires no custom code
- The colors are currently at an 8.59:1 contrast ratio (meeting certain accessibility requirements)

Custom css should be used sparingly

**Describe alternatives you've considered**
- Creating a redline for each page. This is time consuming for the design team
- Implementing the CfA style guidelines. This is time consuming for the frontend team

**Additional context**
The wireframes outline the structure of the frontend. However, they are not intended as a pixel-by-pixel request for design and styling. That is the role of a "redline".

**Labels**
- [x] Add a status label
  - `backlog`
  - `discussing`
  - `ready`
  - `implementing`
- [x] Add additional labels as needed

### For Developers
**Affected Components**
app.css

**Technical Resources**
Add links to appropriate documentation. Describe possible code implementations.

**Design Documentation**
Wireframes or other design resources

**Testing Outline**
Describe the testing methods appropriate for this feature

**Comment by TangoYankee** [comment]
I took the idea of wireframes =/= redlines and applied it retroactively. I eliminated all the custom styling that was made based on the wireframes. The results are below and look like the best option to me. I also changed the html header tags to follow an ascending pattern, starting with the navigation title as h1. This is for accessibility considerations. The code is available at [ty/minimum-custom-styling](https://github.com/openoakland/woeip/tree/ty/minimum-custom-styling)
@nbilenko @theecrit 

![WOAQ login 2019-04-20](https://user-images.githubusercontent.com/11068062/56460257-bd8c4500-6354-11e9-8997-0e430b90f513.png)
